frontend/src/app/navbar/navbar.component.ts

```diff
@@ -24,9 +24,9 @@ import { NgbDropdown, NgbDropdownItem, NgbDropdownMenu, NgbDropdownToggle } from
   standalone: true,
   imports: [RouterLink, FontAwesomeModule, TranslateModule, NgbDropdown, NgbDropdownToggle, NgbDropdownMenu, NgbDropdownItem]
 })
-export class NavbarComponent implements OnInit {
+export class NavbarComponent {
   collapsed = true;
-  user: User | null | undefined = undefined;
+  user: User | null = null;
```

Comment:
No I think it's useful to distinguish between "no user" and "we don't know yet", because the oauth-based login can take some time before emitting.
But I agreen that this is not clear.
Maybe we could, instead, use this to make it clearer:
```
type ViewModel = { status: 'unknown' | 'absent'; user: null; } | { status: 'present', user: User; };
```
and then do

```
  vm!: ViewModel;
  
  constructor(private authenticationService: AuthenticationService) {
    this.authenticationService.getCurrentUser().pipe(
      map(user => user ? { status: 'present', user } : { status: 'absent', user: null }),
      startWith({ status: 'unknown', user: null })
    ).subscribe(vm => this.vm = vm);
  }
```

Of course, we'll have to adapt the template.